kubetest2-tf/deployer/deployer.go

```diff
@@ -131,13 +131,18 @@ func (d *deployer) Up() error {
 
 			if oerr != nil {
 				if i == retryOnTfFailure {
-					return fmt.Errorf("terraform.Output failed: %v", oerr)
+					klog.Infof("terraform Apply and terraform Output failed. Look into it and delete the resources")
```

Comment:
oops, lets have a flag to make this decision :) lets not exit by default.

Reply:
Done.